packages/app-extrinsics/src/Extrinsic.tsx

```diff
@@ -82,7 +88,9 @@ class Extrinsic extends React.PureComponent<Props, State> {
           isValid &&
           !isUndefined(values[index]) &&
           !isUndefined(values[index].value) &&
+          isValidLength(values, index) &&
```

Comment:
The component would have already done this, i.e. it cannot be `isValid` and then have an invalid length - `isValid` for the type should indicate that the value is valid and passes all checks, including length (if applicable).